akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileUploadDirectivesSpec.scala

```diff
@@ -147,7 +148,7 @@ class FileUploadDirectivesSpec extends RoutingSpec {
   }
 
   private def read(file: File): String = {
-    val in = new FileInputStream(file)
+    val in = Files.newInputStream(file.toPath)
```

Comment:
Note that [this line](https://github.com/akka/akka-http/pull/1038/files#diff-e1d2ed324b8dda05ffd842932a01ce76L154) could corrupt the input if the last byte(s) are an unclosed UTF-8 sequence. (Also, I think it would be faster to pass in the [UTF-8](https://docs.oracle.com/javase/7/docs/api/java/nio/charset/StandardCharsets.html#UTF_8) Charset rather than the "UTF-8"-String there.